lib/event.js

```diff
@@ -687,6 +702,15 @@ var ICalEvent = function(_data, calendar) {
                 g += ';BYMONTHDAY=' + data.repeating.byMonthDay.join(',');
             }
 
+            if(data.repeating.exclude) {
+                g += ';EXDATE=';
```

Comment:
This will output EXDATE as a parameter of RRULE, like `RRULE:FREQ=<x>;EXDATE=<x>`.  From reading the spec EXDATE should be its own property (line).  I'd recommend writing a test for the behaviour that you want and making sure it passes.